pdh_json_updater/json_card.py

```diff
@@ -32,6 +31,15 @@ def __init__(self, scryfall_queried_card):
         self.legality = JsonCard.is_legal(scryfall_queried_card)
         self.isPauperCommander: bool = JsonCard.is_commander(scryfall_queried_card)
 
+    def __json__(self):
```

Comment:
Methods with names formatted like this are called "magic methods" and they do some extra weird stuff sometimes, I would just call this `to_json` to avoid any accidental weirdness